node/consensus/Cargo.toml

```diff
@@ -57,7 +57,7 @@ version = "=2.2.7"
 path = "../bft/ledger-service"
 version = "=2.2.7"
 default-features = false
-features = [ "ledger", "ledger-write" ]
+features = [ "ledger", "ledger-write", "test" ]
```

Comment:
it'd probably be better if we didn't use `test` features in non`-dev` dependencies; we should probably make the required items available outside of test scenarios instead

Reply:
Good catch, I only need it in the test scope: 38e973002